index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<main id="main">
+    <h1 id="title">Dr Norman borlaugh</h1>
+    <p>The man who saved a billion lives
+
+    </p>
+    <div id="img-div">
+        <img  id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="pic of dr">
```

Comment:
I suggest using external CSS in which you can use the same .css file for multiple pages. But using inline CSS can increase the page’s size and loading time.